backend/python/app/models/story_translation.py

```diff
@@ -10,10 +10,11 @@
 
 
 class StoryTranslation(db.Model):
+
     __tablename__ = "story_translations"
     id = db.Column(db.Integer, primary_key=True, nullable=False)
     story_id = db.Column(db.Integer, db.ForeignKey("stories.id"), nullable=False)
-    language = db.Column(pgEnum(LanguageEnum), nullable=False)
```

Comment:
Why change to string column?

Reply:
SQLAlchemy and Graphene's enum types aren't implicity castable so trying to use an enum here and in SQLAlchemy makes the code not work. I saw this approached used on the stories model (albeit for slightly different reasons) so I'm not 100% sure what the call was here, forgot to include this in the intial pr sorry :/

Reply:
hmm interesting, enums are pretty wack. It might get even weirder when we switch to mysql so keeping this as a string for now is fine too. ty for explaining :)